src/Contract/CborBytes.purs

```diff
@@ -0,0 +1,16 @@
+-- | A module with CBOR-related functionality.
+module Contract.CborBytes (module CborBytes) where
+
+import Ctl.Internal.Types.CborBytes
+  ( CborBytes(CborBytes)
+  , byteLength
```

Comment:
Let's not export functions more than once from `Contract.*` modules. This one is in `Contract.Prim.ByteArray`.

Reply:
It is not exactly the same, but it does use `Contract.Prim.ByteArray` inside the function: https://github.com/Plutonomicon/cardano-transaction-lib/blob/014e2b0119a16bd5ca936ac28d26ffeea8e9db35/src/Internal/Types/CborBytes.purs#L61-L62

We can be easily remove from the export list, as it is a simple function. Or maybe rename it to `cborByteLength`?

Reply:
OK I didn't notice it's different

Reply:
Yes, a rename would be good.